sdks/go/examples/minimal_wordcount/minimal_wordcount.go

```diff
@@ -53,6 +53,9 @@ import (
 var wordRE = regexp.MustCompile(`[a-zA-Z]+('[a-z])?`)
 
 func main() {
+    // beam.Init() is an initialization hook that must be called on startup.
+    beam.Init()
```

Comment:
Please run gofmt on the modified files, the indentation is off.

You should be able to configure your IDE to run gofmt on save for .go files. Most Go plugins should be able to handle this too.

Reply:
Done, applied gofmt and squashed the fixup commit in.